src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/extensions */
+
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const operandOne = Big(numberOne);
+  const operandTwo = Big(numberTwo);
+
+  switch (operation) {
+    case 'X':
+      return operandOne.times(operandTwo).toString();
+    case '-':
+      return operandOne.minus(operandTwo).toString();
+    case '+':
+      return operandOne.plus(operandTwo).toString();
+    case '%':
+      return operandOne.mod(operandTwo).toString();
```

Comment:
In the calculator, this is not the modulus operator, it should be a modifier turning the number into a percentage based number.